applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -1356,58 +1412,60 @@ int unicast_client_config_get(struct bt_conn *conn, enum bt_audio_dir dir, uint3
 		return -ENXIO;
 	}
 
-	ret = device_index_get(conn, &device_idx);
+	ret = device_index_get(conn, &idx);
 	if (ret) {
 		LOG_WRN("No configured streams found");
 		return ret;
 	}
 
```

Comment:
`unicast_servers[idx.lvl1][idx.lvl2][idx.lvl3]` is accessed six times in this function, so consider a variable